lib/iris/cube.py

```diff
@@ -64,7 +65,8 @@ class CubeMetadata(collections.namedtuple('CubeMetadata',
                                            'var_name',
                                            'units',
                                            'attributes',
-                                           'cell_methods'])):
+                                           'cell_methods',
+                                           'fill_value'])):
```

Comment:
I have explicitly commented on this
https://github.com/SciTools/iris/issues/2394#issuecomment-281644724

but without details

there are test problems if we try to promote shape to be cube metadata.  I'd like to hold off on doing this for now and stick with shape being inferred

i'll need to dig out the tests which caused problems, but it unravelled on me, so i left it out for now

to be continued